TaskManager/Modules/Home/View/HomeViewController.swift

```diff
@@ -7,52 +7,76 @@
 
 import UIKit
 
-enum Section: String, CaseIterable {
+enum Section: String {
     case Active
     case Completed
+    
+    func localizedEnum() -> String {
+        switch self {
+        case .Active:
+            return LocalizableString.Home.activeSection
+        case .Completed:
+            return LocalizableString.Home.completedSection
+        }
+    }
 }
 
 protocol HomeView: AnyObject {
     func showTask(tasks: [ModelTask])
+    func appendItems(task: ModelTask, section: Section)
 }
 
-class HomeViewController: UIViewController, UITableViewDelegate, HomeView {
+class HomeViewController: UIViewController, HomeView {
     
     @IBOutlet weak private var editButton: UIButton!
     @IBOutlet weak private var taskLabel: UILabel!
     @IBOutlet weak private var tableView: UITableView!
     @IBOutlet weak private var addButton: UIButton!
     
     private var dataSource: HomeDataSource!
+    private var snapshot: NSDiffableDataSourceSnapshot<Section, ModelTask>!
     var presenter: HomeViewPresenter!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         if #available(iOS 15.0, *) {
             tableView.sectionHeaderTopPadding = 0.0
-        } 
+        }
+        localizedString()
```

Comment:
no need to call this function localizedString a name like setupUI or configUI would be enough in this case